config/image_optim.yml

```diff
@@ -1,12 +1,12 @@
 skip_missing_workers: true
 pngout: false
-pngcrush: false
-advpng: false
-optipng: false
-pngquant: false
-jhead: false
-jpegoptim: false
-gifsicle: false
-jpegtran: false
+pngcrush: true
+advpng: true
+optipng: true
+pngquant: true
+jhead: true
+jpegoptim: true
+gifsicle: true
+jpegtran: true
```

Comment:
It's probably easier (and more consistent) just to delete the lines - they all default to on anyway.

Reply:
Indeed. Pushed.